src/Bicep.Core.UnitTests/PrettyPrint/PrettyPrinterValidSyntaxTests.cs

```diff
@@ -226,32 +227,32 @@ public class PrettyPrinterValidSyntaxTests : PrettyPrinterTestsBase
   false
 ]",
 // Formatted.
-@"param foo object = { // I can be anywhere
-}
+@"param foo object = {// I can be anywhere
+  }
```

Comment:
It looks like the whitespace got moved to the other side of the comment.

Reply:
Initially I found it to be a tricky edge case to fix in the v1 formatter, and I was going to fix it in the v2 formatter. I tried harder this time and found a simple solution, which happen to fix https://github.com/Azure/bicep/issues/9862 as well :). See changes here: https://github.com/Azure/bicep/pull/10036/commits/af8def46e6c2c7e8a4b783330fd2958f20c27c64#diff-8db62c00ecc59db1369f7708689de4cd5aa38f21fe041f9d02b286b6288490a3.